ocaml/xapi/xapi_fist.ml

```diff
@@ -29,6 +29,11 @@ let fistpoint name =
     true
   with _ -> false
 
+let rec hang_while fp =
```

Comment:
Maybe maintain a counter and emit a log line if `counter mod 10 = 0`? This way you can observe from the outside what is happening